chart/openfaas/templates/controller-rbac.yaml

```diff
@@ -0,0 +1,101 @@
+{{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
+{{- if eq .Values.operator.create false }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: {{ template "openfaas.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: faas-controller
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Release.Name }}-controller
+  namespace: {{ .Release.Namespace | quote }}
+{{- if .Values.rbac }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
Is this the ClusterRole for faas-netes? We didn't need to use ClusterRole in the plain YAML so this should probably run with least privileges too. The work is was tested via #212 but not merged.

Reply:
We can merge this as it is and ask for a rebase on #212.